package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "leaderboard",
+  "description": "",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --mode development",
+    "build": "webpack --mode production"
+  },
```

Comment:
- [ ]  I try running npm run start after installing dependencies by running `npm install` the app is not opened in the browser. I checked your package.json file and found out that the start script which is responsible for starting your app in the browser is missing, I added the start script "start": "webpack serve --open" and I was able to run your app with npm run start but I  got the error below :point_down: 

![badweel](https://user-images.githubusercontent.com/57562869/156324406-69e2b2bd-18e8-46b0-a5ec-3847801cbda7.png)

- But don't worry to solve this got to your webpack.config.js and add this :point_down: 
`mode: 'production', ` or `mode: 'development',`